features/user_can_navigate_to_a_specific_restaurant_page.feature

```diff
@@ -9,8 +9,10 @@ Feature: Visitor can view details about a specific restaurant
     | ThaiTanic       | Thailands finest food, watch out for the iceberg |
     | Fu King Chinese | Best chinese food in town                        |
 
-  Scenario: Visitor visits a specific restaurant page
+  Scenario: Visitor navigates to a specific restaurant page
     Given I visit the landing page
+    And I fill in "assortment" with "Thai"
+    And I click "Search"
```

Comment:
Why are we testing this here and in the other feature?

Reply:
Mostly because we needed to change the other feature too since it was still related to have the restaurants hardcoded on the page, we might delete some old features or make them into one since we're testing the same things in different old features

Reply:
We removed two obsolete features since we're testing them  already in the assortment acceptance test